tools/integrations/manage.sh

```diff
@@ -195,10 +197,12 @@ cmd_publish() {
   fi
 
   # setting local variables for docker image versioning
+  local docker_user="airbytebot"
```

Comment:
The bot name is hard-coded in the Github action, and doesn't seem to be available via ENV/secrets.  Moving this to ENV could be future work.